bakerydemo/static/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
```

Comment:
Leaving this in here feels like a potential footgun for a beginner learning from this project.